src/main/webapp/app/tasks/task-search/task-search.component.scss

```diff
@@ -0,0 +1,9 @@
+  
```

Comment:
be wary of unecessary spaces and new lines

Reply:
I removed it. Thanks for hints